ada/ast.py

```diff
@@ -94,6 +94,51 @@ class AdaNode(ASTNode):
         public=True
     )
 
+    @langkit_property(ignore_warn_on_node=True)
+    def withed_unit_helper(unit_name=T.Name):
+        """
+        Static method helper. Fetch the unit designated by unit_name. Return
```

Comment:
I think it's worth mentioning what else this method does compared to the previous property that had this exact documentation. (i.e. it emits an error when it can detect for sure that a unit is missing?)